report_thesis/src/sections/results/initial_results.tex

```diff
@@ -31,7 +32,10 @@ \subsection{Initial Results}
 Although \gls{xgboost} and \gls{svr} appear the most frequently in Table~\ref{tab:best_model_occurrences}, this does not imply that they are the best models for every oxide.
 The tables also demonstrates the importance of judging model performance per oxide.
 For example, if one were to only consider the mean of the performance metrics, \gls{pls} would be considered among the worst performing models, as shown in Figure~\ref{fig:init_results_rmses}.
-However, inspecting Table~\ref{tab:best_results} reveals that \gls{pls} exhibits the lowest \gls{rmsecv} and standard deviation of prediction errors for both \ce{MgO} and \ce{Na2O}, indicating that it is the most accurate and robust model for these oxides.
+However, inspecting Table~\ref{tab:best_results} reveals that \gls{pls} exhibits the lowest \gls{rmsecv} and standard deviation of prediction errors for both \ce{MgO} and \ce{Na2O}.
+
```

Comment:
Probably don't want  the linebreak when you're referencing the oxides in the sentence below?
```suggestion
```